content/riak/ts/1.2.0/using/querying.md

```diff
@@ -200,7 +200,7 @@ The following operators are supported for each data type:
 
 #### Quanta query range
 
-A query covering more than a certain number of quanta (5 by default) will generate too many sub-queries and the query system will refuse to run it. Assuming a default quanta of 15 minutes, the maximum query time range is 75 minutes.
+A query covering more than a certain number of quanta (5 by default) will generate the error `too_many_subqueries` and the query system will refuse to run it. Assuming a default quanta of 15 minutes, the maximum query time range is 75 minutes.
```

Comment:
Should this change (and the other on this page) be back-ported to 1.1.0 and 1.0.0?